### DISCUSSION TITLE
Advice needed on managing and storing duplicate data

### DISCUSSION THREAD

**Discussion by Olindn** [statamic/cms#7370] [General]
Hi,

I am planning to create a new website for a phone repair company using Statamic Pro (really excited for this!) and I am looking for some advice. 

The website will have a list of all phone brands and their respective models. Each phone will have multiple services, such as LCD screen replacement, battery replacement, and camera replacement. Keeping the prices updated and easily manageable is a concern. I am considering using a custom control panel view to manage all phones of one brand on one page, instead of having separate pages for each phone.

Additionally, each service needs a description. In most cases, battery replacement services do not require different descriptions for each phone. To keep the site manageable, I would like to store a general description and provide a different description for a particular phone if needed. I have no clue how to manage this.

Would appreciate your thoughts.

Thanks,
Omid

**Comment by mscruse** [comment]
I've done a similar thing with a "Call to action" on a website. What I did was in my globals I have the default call to action text and in my pages blueprint the ability to override the default - a toggle to indicate you want to override then the field(s) needed to override.

The output will use the default unless otherwise set per page.

This way you can have all your defaults in once place.

Does that help?
